packages/adaptive_scaffold/example/lib/main.dart

```diff
@@ -0,0 +1,656 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:adaptive_scaffold/adaptive_scaffold.dart';
+import 'package:flutter/material.dart';
+
+
+// A more functional demo of the usage of the adaptive layout helper widgets.
+// Modeled off of the example on the Material 3 page regarding adaptive layouts.
+// For a more clear cut example usage, please look at adaptive_layout_demo.dart
+// or adaptive_scaffold_demo.dart
+
+void main() {
```

Comment:
Other than the one above, there are no comments in the entire file.  Since this is an example, it would be good to have some commentary about what each section is about.  Doesn't need to be a book, just enough to follow what's going on if you've never seen it before.